compiler/optimizer/ValuePropagationCommon.cpp

```diff
@@ -2371,7 +2372,7 @@ void OMR::ValuePropagation::generateArrayTranslateNode(TR::TreeTop *callTree,TR:
    TR::Node *srcOff = srcOffNode->createLongIfNeeded();
    TR::Node *lenNode = NULL;
    TR::Node *len = NULL;
-   if (!isISO88591Encoder)
+   if (!(isISO88591Encoder || isAsciiEncoder))
```

Comment:
Just a suggestion, do you think this has better readability compared what you have now? 
```suggestion
   if (!isISO88591Encoder  && !isAsciiEncoder)
```

Reply:
I personally don't have a preference in this scenario. I previously used to write them the way you suggested above. 

However in my previous PRs reviewers have suggested adding `()` around negation in `if` statements. I think the idea there was that in `if (!isISO88591Encoder)` it's easy to miss the `!` for someone skimming through the code.

If you prefer your approach, then I'll revert back to that. Let me know.

Reply:
Either way works. Just had a thought, you can keep the code as it is. But please add spaces at appropriate places.